src/main/scala/org/apache/spark/deploy/armada/submit/ArmadaUtils.scala

```diff
@@ -53,4 +53,8 @@ object ArmadaUtils {
             sleep 1;
           done
         """.stripMargin.trim
+
+  def getExecutorRange(numberOfExecutors: Int): Range = {
+    0 until numberOfExecutors
```

Comment:
I think it is clearer if you just inline this

Reply:
That was actually the bug that I fixed, so I would prefer to keep it like this.

You inlined it with "0 until" and I inlined it with "1 to"  and then the executer tracker incorrectly detected a failure and stopped the driver.

Reply:
This way it can easily be unit tested. (This is unit tested, right?)

Reply:
unit tests are in the changes for this file: src/test/scala/org/apache/spark/scheduler/cluster/armada/ArmadaClusterManagerBackendSuite.scala